bears/vcs/git/GitCommitBear.py

```diff
@@ -123,6 +123,9 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
                 yield Result(self, 'HEAD commit has no message.')
             return
 
+        if shortlog.startswith('Revert') and len(body) == 0:
```

Comment:
So you are not detecting revert commits if they do have a reason?

Reply:
I don't think revert commits with reason have any problem.

Reply:
When revert commits don't have any reason then they are wrong. Hence, detected only then.

Reply:
I am not sure what needs to be done here. On what was meant by the issue, whether you have to detect revert commits and an additional check if they don't have a reason, or for only when they don't have a reason. You should discuss this with someone

Reply:
I did mention @sils once on gitter asking something like this. Never got any response.

Reply:
you can discuss with any other maintainer 👍

Reply:
Okay. I will write tests for this first and then discuss about detecting revert commits in general. Thanks :)

Reply:
Then you need to change tests again accordingly, so its better to ask someone what to do first ;)

Reply:
I don’t particularly like this solution since reason could be given in the shortlog as well, and failing coala on a repository due to that is not good.
Also, is there a better way to detect reverting of commits? I couldn’t find one myself, but it seems this might over-trigger quite easily.

Reply:
I would also say: Simple by default, powerful if needed.
We should implement other settings controlling what is considered a revert commit. But that can be part of a new issue.

Reply:
So I would remove the `len(body)` part

Reply:
And btw: I want to see a setting to enable looking for revert-commits :+1: